DFC.Digital/DFC.Digital.Service.CourseSearch/Services/Contracts/IConvertTribalCodes.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace DFC.Digital.Service.CourseSearchProvider
+{
+    public interface IConvertTribalCodes
+    {
+        string[] GetTribalAttendanceModes(string attendanceMode);
```

Comment:
Why can't this return `IEnumerable<string>`

Reply:
End consumer needs its as an array.. so either we get as this as such  or we will convert later to array